readme.md

```diff
@@ -451,6 +451,12 @@ const instance = got.extend({
 
 The response object will typically be a [Node.js HTTP response stream](https://nodejs.org/api/http.html#http_class_http_incomingmessage), however, if returned from the cache it will be a [response-like object](https://github.com/lukechilds/responselike) which behaves in the same way.
 
+##### request
+
+Type: `Object`
+
```

Comment:
It'd be nice to have a note here that it is *not* [`http.ClientRequest`](https://nodejs.org/api/http.html#http_class_http_clientrequest).